### ISSUE TITLE
Cross Op

### ISSUE THREAD

**Issue by mtngld** [onnx/onnx#2683]
Currently, ONNX  does not include the Cross op.

PyTorch cross:
https://pytorch.org/docs/stable/torch.html#torch.cross

TF cross:
https://www.tensorflow.org/api_docs/python/tf/linalg/cross

**Comment by ebarsoum** [comment]
Which model need it?

**Comment by mtngld** [comment]
It is used often for geometric related models, for example in this pytorch_geometric library

https://github.com/rusty1s/pytorch_geometric

https://github.com/rusty1s/pytorch_geometric/blob/5692a8613d9340c76545e59b4d37cd4ced6d340e/torch_geometric/nn/conv/ppf_conv.py#L11

**Comment by vegetable09** [comment]
@mtngld I met the same problem and work around it by the following code
```
        # normal = tu.cross(tv, dim=3) 
        normal = torch.stack((
        tu[:,:,:,1]*tv[:,:,:,2] - tu[:,:,:,2]*tv[:,:,:,1],
        tu[:,:,:,2]*tv[:,:,:,0] - tu[:,:,:,0]*tv[:,:,:,2],
        tu[:,:,:,0]*tv[:,:,:,1] - tu[:,:,:,1]*tv[:,:,:,0]), dim=3)
        # print("cross all close", torch.allclose(normal, normal2))
```
BTW, @ebarsoum cross is frequently used by getting normal in the 3d or rendering tasks. And the CVPR 2020 best paper unsup3d, and ECCV 2020 best paper are both about 3d and rendering. I'd appreciate if onnx will support cross operator ASAP

**Comment by saeed1262** [comment]
Any chance this will be added to the supported ops?

**Comment by ivan-alles** [comment]
I need this too, to computing elements of a rotation matrix.

**Comment by justinchuby** [comment]
https://github.com/microsoft/onnx-script/pull/512

**Comment by Francis235** [comment]
I need this op too, is this operator supported now?

**Comment by andife** [comment]
Being added to onnxscript means that the export should work for Pytorch? 
But not a separate operator but a combination of other operators?

**Comment by justinchuby** [comment]
> Being added to onnxscript means that the export should work for Pytorch? But not a separate operator but a combination of other operators?

That is correct. I am not aware of a strong motivation to standardize this operator in the ONNX spec
